.for_maintainers/release/RELEASE_NOTES_TEMPLATE.md

```diff
@@ -0,0 +1,167 @@
+# Release Notes
+
+Overview of release and what is included.
+
+## Table of Contents
+1. [Added](#added)
+    - [example](#example)
+2. [Changed](#changed)
+3. [Fixed](#fixed)
+4. [Upgrading Wallaroo](#upgrading-wallaroo)
+    - [Upgrading Wallaroo when compiled from source](#upgrading-wallaroo-when-compiled-from-source)
+    - [Upgrading Wallaroo via Wallaroo Up](#upgrading-wallaroo-up)
+    - [Upgrading Wallaroo in Docker](#upgrading-wallaroo-in-docker)
+    - [Upgrading Wallaroo in Vagrant](#upgrading-wallaroo-in-vagrant)
+5. [CHANGELOG](#changelog)
+
+<a name="added"></a>
+## Added
+
+<a name="example"></a>
+### Example
+
+<a name="changed"></a>
+## Changed
+
+<a name="fixed"></a>
+## Fixed
+
+<a name="upgrading-wallaroo"></a>
+## Upgrading Wallaroo
+
+In all cases below, if you run into issues, please reach out to us! We’re available on [twitter](https://twitter.com/wallaroolabs), [IRC](https://webchat.freenode.net/?channels=#wallaroo), [Github](https://github.com/WallarooLabs/wallaroo), [by email](mailto:hello@wallaroolabs.com), or [our mailing list](https://groups.io/g/wallaroo).
+We love questions!
+
+**If you have made no changes to Wallaroo or Pony since installation, your best bet will be to delete your Wallaroo installation and start from scratch, following the [instructions] (https://docs.wallaroolabs.com/book/getting-started/choosing-an-installation-option.html) of your choice.**
```

Comment:
Is this line still required now that we're using wallaroo version specific directories as part of the installation process?

Reply:
will update